contrib/systemd/README.md

```diff
@@ -26,7 +29,9 @@ You can refer to [this example](https://github.com/containers/podman/blob/main/c
 Assuming the status messages show no errors, the libpod service is ready to respond to the APIv2 on the unix domain socket `/run/user/$(id -u)/podman/podman.sock`
 
 ### podman.service
-You can refer to [this example](https://github.com/containers/podman/blob/main/contrib/systemd/user/podman.service) for a rootless podman.service file.
+You can refer to [this example](https://github.com/containers/podman/blob/main/contrib/systemd/system/podman.service.in) for a rootless podman.service file.
+
+Note: replace `@@PODMAN@@` with path to podman binary, such as `/usr/local/bin/podman`
 
 ### podman.socket
-You can refer to [this example](https://github.com/containers/podman/blob/main/contrib/systemd/user/podman.socket) for a rootless podman.socket file.
+You can refer to [this example](https://github.com/containers/podman/blob/main/contrib/systemd/system/podman.socket) for a rootless podman.socket file.
```

Comment:
Given they are duplicated from the above I feel like we should drop this entirely